app/backend/src/couchers/models.py

```diff
@@ -1876,9 +1880,61 @@ def is_leaf(self) -> bool:
             unique=True,
             postgresql_where=is_official_cluster,
         ),
+        # trigram index on unaccented name, only for official clusters
+        Index(
+            "idx_clusters_name_unaccented_trgm",
+            name_unaccented,
+            postgresql_using="gin",
+            postgresql_ops={"name_unaccented": "gin_trgm_ops"},
+            postgresql_where=is_official_cluster,
+        ),
```

Comment:
You do not need to add a new column in the table for this (also triggers are in general a last-resort solution due to how messy they are), you can create an index on a generated column. So I think you just have to replace `name_unaccented` with `func.lower(func.unaccent(name))` or similar.

Also no need to restrict to only official clusters.